include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -354,33 +354,6 @@ struct __get_sycl_range
     //We have to keep sycl buffer(s) instance here by sync reasons;
     ::std::vector<::std::shared_ptr<oneapi::dpl::__internal::__lifetime_keeper_base>> m_buffers;
```

Comment:
For which reasons we define `m_buffers` based on `std::shared_ptr` ?
Looks like it's enough to define through `std::unique_ptr`...

Reply:
cannot call to mind why `std::shared_ptr`.... we don't share ownership if such buffer, just keep ones within __get_sycl_range lifetime.

Reply:
May you change it to `std::unique_ptr` or may be remove as proposed @danhoeflinger [here](https://github.com/oneapi-src/oneDPL/pull/1254#discussion_r1380057764)?